apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-title/edit.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Do we really need to suppress this rule? I'm not getting any linter error if I remove this line.